dp/cloud/go/services/dp/storage/dp_manager.go

```diff
@@ -0,0 +1,35 @@
+package storage
+
+import (
+	"database/sql"
+
+	"magma/orc8r/cloud/go/sqorc"
+
+	sq "github.com/Masterminds/squirrel"
+)
+
+type DpSqlManager interface {
```

Comment:
I think we don't need to define this interface as it is unused

Reply:
done